cherrypy/test/test_logging.py

```diff
@@ -216,7 +216,7 @@ def test_utc_in_timez(monkeypatch):
 
     class mock_datetime:
         @classmethod
-        def utcnow(cls):
+        def now(cls, _):
```

Comment:
Could you still name the argument, so the signature matches?
```suggestion
        def now(cls, tz):
```

If flake8 is unhappy, that would warrant an inline `# noqa: <error-code>` with a code comment explaining it.

Reply:
Done

Reply:
Thanks! Marking this as resolved!